README.md

```diff
@@ -8,46 +8,67 @@ A service mesh for polyglot microservices.
+   services. The proxies form a _secure Layer-7 microservice mesh_
+   providing a rich set of functions like discovery, rich layer-7 routing,
+   circuit breakers, policy enforcement and telemetry recording/reporting
+   functions.
+
+* **Mixer** - Central component that co-ordinates with various proxies to
+   enforce policies such as ACLs, rate limits, authentication, request
+   tracing and metrics collection.
+
+* **Manager** - A configuration manager responsible for configuring the
+  proxies and the mixer at runtime.
+
+A high-level overview of various components in Istio is available
+[here](doc/overview.md). In terms of platforms, Istio currently supports
+Kubernetes. We plan to add support for additional platforms in the near
+future. See the [getting started](doc/getting-started.md) tutorial for more
+information on using Istio in your Kubernetes deployments.
 
 ## Repositories
 
 The Istio project is divided across multiple GitHub repositories. Each
 repository contains information about how to build and test it.
 
 - [istio/api](https://github.com/istio/api). This repository defines
-component-level APIs and common configuration  formats for the Istio platform.
-
-- [istio/istio](https://github.com/istio/istio). The main Istio repo which is
-used to host the high-level documentation for the project, along with 
-examples & demos.
-
-- [istio/manager](https://github.com/istio/manager). The Istio manager is 
-used to configure Istio and propagate configuration to the other components 
-of the system, including the Istio mixer and the Istio proxy mesh.
-
-- [istio/mixer](https://github.com/istio/mixer). The Istio mixer is the nexus of
-the Istio service mesh. The proxy delegates policy decisions to the mixer, and
-both the proxy and Istio-managed services direct all telemetry data to the
-mixer. The mixer includes a flexible plugin model enabling it to interface to
-a variety of host environments and configured backends, abstracting the 
-proxy and Istio-managed services from these details.
+component-level APIs and common configuration formats for the Istio platform.
+
+- [istio/istio](README.md). This is the main Istio repo (the one you are
+currently looking at). It hosts the high-level documentation for the
+project, along with [tutorials](doc/getting-started.md) and two demo
+applications: a [basic echo app](demos/apps/simple_echo_app) and a slightly
+more advanced [polyglot application](demos/apps/bookinfo).
+
+- [istio/manager](https://github.com/istio/manager). This repository
+contains platform specific code to populate the
+[abstract service model](doc/model.md), dynamically reconfigure the proxies
+when the application topology changes, as well as translate
+[routing rules](doc/rule-dsl.md) into proxy specific configuration.  The
+[_istioctl_](doc/istioctl.md) command line utility is also available in
+this repository.
+
+- [istio/mixer](https://github.com/istio/mixer). The mixer repository
```

Comment:
The idea being we don't have to talk about design rationale in the repo description.